src/components/Compare/Compare.style.tsx

```diff
@@ -396,27 +412,24 @@ export const HeaderWrapper = styled.div`
   padding: 0.75rem;
 `;
 
-export const Header = styled.div`
+export const Header = styled.div<{ isHomepage?: boolean }>`
   display: flex;
   font-family: Roboto;
   font-weight: bold;
   font-size: 1.5rem;
+  margin: ${({ isHomepage }) => isHomepage && '0 auto'};
```

Comment:
nit: consider renaming the prop `centered` so it's a bit less context-dependent.